apps/prairielearn/src/lib/preact.tsx

```diff
@@ -4,7 +4,7 @@ import { render } from 'preact-render-to-string/jsx';
 import { compiledScriptPath, compiledScriptPreloadPaths } from '@prairielearn/compiled-assets';
 import { AugmentedError } from '@prairielearn/error';
 import { type HtmlSafeString, html } from '@prairielearn/html';
-import { Fragment, type VNode } from '@prairielearn/preact-cjs';
+import { type ComponentChildren, Fragment, type VNode } from '@prairielearn/preact-cjs';
```

Comment:
Nit, with our aliases, this can import straight from `preact`.

```suggestion
import { type ComponentChildren, Fragment, type VNode } from 'preact';
```

ESLint will want to reorder these imports after that change.